source/release-notes.txt

```diff
@@ -17,6 +17,63 @@ Download the latest {+rel-mig+} binary from the `release page <https://www.mongo
 For installation instructions, see the :ref:`Installation <installation>` 
 page. 
 
+1.13.1 Changelog
+----------------
+
+*Released April 29, 2025*
+
+Data Integrity Advisory:
+
+.. important::
+
+   This release includes two critical fixes for data integrity issues affecting 
+   Snapshot and Continuous Migrations as listed below. The Relational Migrator 
+   team has also identified further data integrity issues that affect certain 
+   Continuous Migration configurations. These issues are scheduled to 
+   be addressed shortly in an upcoming release. For safety, this release 
+   temporarily disables Continuous Migrations (CDC mode) for projects which 
+   include any embedded document or array mappings. Potentially affected users 
+   will be provided instructions to override this check.
+
```

Comment:
Here's how we do critical advisories for the Server. This might be a good approach if we are talking about critical advisories rather than a critical bug fix.